web/src/components/Collapsible.tsx

```diff
@@ -53,10 +53,9 @@ const Collapsible = ({
   return (
     <Container className={className}>
       <CollapsibleHeader
-        role='button'
+        ariaLabel={t(collapsed ? 'showMore' : 'showLess')}
         onClick={() => setCollapsed(!collapsed)}
-        tabIndex={0}
-        onKeyUp={() => setCollapsed(!collapsed)}>
```

Comment:
Why have you decided to disable the key handling?

Reply:
The key handling is still there, just now with the default button behavior that only specific keys (e.g. enter) trigger the onClick. This also fixes e.g. this bug: #2572.